ui/raidboss/raidemulator.js

```diff
@@ -18,6 +18,12 @@ import { TimelineLoader } from './timeline';
 import Tooltip from './emulator/ui/Tooltip';
 import UserConfig from '../../resources/user_config';
 import raidbossFileData from './data/manifest.txt';
+// @TODO: For some reason this absolutely has to be imported as `Worker`, no matter what settings
+// are specified in webpack.config.cjs
+// Using `{ Worker as NetworkLogConverterWorker }` also fails for some reason.
+// Using `{}` works for some reason too.
+// eslint-disable-next-line import/default
```

Comment:
Would https://www.npmjs.com/package/eslint-import-resolver-webpack fix this?

Reply:
I tried various options for this and wasn't able to get at least vscode's eslint plugin to recognize the loader. I didn't try running `npm lint` to see if that would pick it up properly since it's about time for me to go to sleep here.

Reply:
If `npm lint` works, then it should work.  I find that vscode lags a little bit in picking up eslint changes, but will eventually get it.

Reply:
Installed package with:
```
npm install eslint-import-resolver-webpack --save-dev
```

Added this config to `.eslintrc.cjs`:
```diff
  'settings': {
    'import/resolver': {
      'node': {
        'extensions': ['.d.ts', '.ts', '.js'],
      },
      'typescript': {
        'alwaysTryTypes': true,
        'project': './tsconfig.json',
      },
+      'webpack': {
+        'config': './webpack/webpack.config.cjs',
+      },
    },
  },
```

```
npm run lint

> cactbot@ lint C:\Users\valar\Source\cactbot2
> eslint **/*.{js,cjs,mjs,ts}


C:\Users\valar\Source\cactbot2\ui\raidboss\raidemulator.js
  21:8  error  No default export found in imported module "./emulator/data/NetworkLogConverterWorker"  import/default
```

Reply:
Yeah, I can't get this to work either.  Maybe it doesn't understand `webpack-loader`.  I think we could write a custom eslint import resolver here, but that's probably overkill for a single line.